src/skills-builder/skills-builder-modal/skillsBuilderModal.scss

```diff
@@ -1,5 +1,9 @@
 .skills-builder-modal {
   button[aria-label="Close"][type="button"]{
-    color: #ffffff;
+    color: $white;
   }
 }
+
+.med-min-height {
+  min-height: map-get($spacers, 6);
+}
```

Comment:
Tangential question: Is this howe we handled similar problems in other MFEs? Is this considered "prior art" for restraining views for smaller screens? Can a similar effect be supported using built-in bootstrap classes? I've always heard in the past that adding Sass/CSS classes should be a last ditch effort. I thought we could also handle breakpoints/sizing through bootstrap (even though I'm not well versed in it).

I know that adding one new CSS class/definition probably isn't a big deal but, as I understand it, we try as best as possible to keep bundle sizes as small as possible, which means as little custom CSS as required.

Reply:
Gotcha! Yeah, this was more for my personal knowledge/learning than "I think this is a problem" :)